frontend/src/types.ts

```diff
@@ -1348,7 +1348,8 @@ export interface FlattenedFunnelStepByBreakdown {
     rowKey: number | string
     isBaseline?: boolean
     breakdown?: BreakdownKeyType
-    breakdown_value?: BreakdownKeyType
+    // :KLUDGE: Data transforms done in `getBreakdownStepValues`
+    breakdown_value?: Array<string | number>
```

Comment:
It's somewhat unclear what "data transforms" are meant here

Reply:
`getBreakdownStepValues` narrows down the type from BreakdownKeyType to always an array - I just made this explicit here. It's a kludge because it's bad behavior but would also require rewriting this view and multiple selectors to solve as it's integral to the app.